AEP/aep-00001.md

```diff
@@ -0,0 +1,177 @@
+# AEP 1 : AEP Purpose and Guidelines
+
+||ArmoniK Enhancement Proposal|
+---:|:---
+**AEP**| 1
+**Title**| AEP Purpose and Guidelines  
+**Author**| Dylan Brasseur <<dbrasseur@aneo.fr>>
+**Status**| Draft
```

Comment:
Rajouter un champs date de création?

Reply:
We can, although this may be redundant with the creation date of the file on GitHub ?